torch/_inductor/fx_passes/post_grad.py

```diff
@@ -1015,6 +1015,29 @@ def fused_int_mm_mul(match: Match, mat1, mat2, mat3, out_dtype=None):
     return inductor.kernel.mm.tuned_fused_int_mm_mul(mat1, mat2, mat3, out_dtype)
 
 
+def is_index_put_and_requires_h2d_sync_for_cuda_value(node):
+    if node.target is not torch.ops.aten.index_put.default:
```

Comment:
do we want this for index_put_ as well ?

Reply:
I figured it would just never get hit since functionalization has already run, and reinplacing runs in a later pass. But probably doesn't hurt to add, can add it